apps/files/src/store.js

```diff
@@ -152,6 +185,40 @@ const actions = {
         console.log('error: ' + file.path + ' not deleted: ' + error)
       })
     }
+  },
+  setFilterTerm (context, filterTerm) {
+    context.commit('SET_FILTER_TERM', filterTerm)
+  },
+  setFileFilter (context, filter) {
+    context.commit('SET_FILE_FILTER', filter)
+  },
+  searchForFile (context, payload) {
+    let client = payload.client
+    let searchTerm = payload.searchTerm
+    context.commit('SET_SEARCH_TERM', searchTerm)
+    // TODO respect user selected listSize from state.config
+    // do not search for empty strings
+    if (!searchTerm) return
+    client.files.search(searchTerm, null, [
+      '{http://owncloud.org/ns}favorite',
```

Comment:
we need to find a common place where all properties are defined and used - this is all currently to much spread around ...

Reply:
You're completlely right, I've got some ideas regarding to that issue; But the needed approach is depending on our webpack setup. I will take a look at the webpack history mode today.

Reply:
hmmm ... I miss the link to webpack .... look at my commit - all dav props are now living in the store